cookiecutter_plugin.json

```diff
@@ -0,0 +1,10 @@
+{
+    "plugin_name": "Name of the plugin",
+    "__plugin_name": "{{ cookiecutter.plugin_name | lower | replace(' ', '') | replace('-', '') | replace('_', '') }}",
+    "plugin_type": ["Exploiter", "Credentials_Collector", "Payload"],
+    "plugin_options": ["y", "n"],
+    "__prompt__": {
+        "plugin_type": "Select the type of plugin",
+        "plugin_options": "Would you like to configure additional options for the plugin?"
```

Comment:
I think this needs to be clearer. Suggestion: "Will your plugin require configuration in order to run?"